Docs/Workflows.md

```diff
@@ -98,4 +98,28 @@ jobs:
 
 If this is for a submodule of [Lombiq's Open-Source Orchard Core Extensions](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/), the `repo` input can be omitted, because the above is its default value. Otherwise, use your parent repository's address in the `{owner}/{repo_name}` format.
 
+## Deploy to Azure App Service
+
+This workflow builds a .NET solution with static code analysis, and then deploys the app to [Azure App Service](https://azure.microsoft.com/en-us/services/app-service/). Example _deploy.yml_:
+
+```yaml
+name: Deploy
+
+on:
+  workflow_dispatch:
+
+jobs:
+  call-deploy-workflow:
+    name: Deploy
+    uses: Lombiq/GitHub-Actions/.github/workflows/deploy.yml@dev
+    with:
+      machine-types: "[\"ubuntu-latest\"]"
+      timeout-minutes: 60
+      app-name: AppName
+      resource-group-name: ResourceGroupName
+      slot-name: Staging
+      url: https://www.myapp.com
+    secrets: inherit
```

Comment:
This passes all the organization secrets to the workflow too. That's a security risk, we only want to pass those secrets that are actually required.